maro/event_buffer/event_buffer.py

```diff
@@ -8,19 +8,14 @@
 #    we can enable this to disable pooling
```

Comment:
@Jinyu-W , confirm with @chaosddp , the event buffer related change need to be merged back first

Reply:
The change already in [PR 197](https://github.com/microsoft/maro/pull/197), waiting to be merged